lib/framework/secrets.dart

```diff
@@ -36,4 +54,9 @@ class SecretsStore with Invokable {
   Map<String, Function> setters() {
     return {};
   }
+
+  _cacheSecret(String key, dynamic value) {
+    secretCache[key] = value;
+    StorageManager().writeSecurely(key: key, value: value.toString());
```

Comment:
why store to storage for?

Reply:
For the ensemble framework code to be able to access app secrets, e.g. auth